pkg/trie/child_storage.go

```diff
@@ -37,8 +37,21 @@ func (t *Trie) SetChild(keyToChild []byte, child *Trie) error {
 	return nil
 }
 
+func (t *InMemoryTrie) getInternalChildTrie(keyToChild []byte) (*InMemoryTrie, error) {
+	key := make([]byte, len(ChildStorageKeyPrefix)+len(keyToChild))
+	copy(key, ChildStorageKeyPrefix)
+	copy(key[len(ChildStorageKeyPrefix):], keyToChild)
+
+	childHash := t.Get(key)
+	if childHash == nil {
+		return nil, fmt.Errorf("%w at key 0x%x%x", ErrChildTrieDoesNotExist, ChildStorageKeyPrefix, keyToChild)
+	}
+
+	return t.childTries[common.BytesToHash(childHash)], nil
+}
+
```

Comment:
seems that this method is a copy of the `GetChild` method.

Reply:
Thanks! was an issue with the merge with dev, solved it